section-engineering-education/engineering-education
# Add resume link, logic and icon.

**Changes:**
- Adds resume link to author frontmatter & logic in `/authors/single.html`.
- Adds resume icon svg.
- Adds instructions to `new_contributors/enged-author-profile-guidelines.md`.

**How to Test:**
Navigate to [beta.section.io/engineering-education/authors/ahmad-mardeni/](https://beta.section.io/engineering-education/authors/ahmad-mardeni/). Ensure that the resume link displays under the students Skills section. It should match the mockup below. This students resume link should navigate to www.google.com. This is just for testing purposes and will be removed before merging to master. The resume link should not appear for other students without the information in their frontmatter. 

<img width="1451" alt="image" src="https://user-images.githubusercontent.com/15935329/182246364-5b9c5e4f-b677-438e-883d-6e123d7369b6.png">
